src/main/java/ConversationManager.java

```diff
@@ -0,0 +1,91 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
```

Comment:
Good job importing only the necessary libraries instead of doing import java.util.*